### ISSUE TITLE
Import JSON does not work

### ISSUE THREAD

**Issue by NesCafe62** [admon84/screeps-room-planner#2]
Steps to reproduce:
1. Place a spawn anywhere
2. Press "Get room JSON" button and copy json to clipboard
3. Reopen the planner
4. Press "Import JSON" button and paste json, then press "Import JSON"
Expected result: Correctly leaded structures plan
Actual result: Nothing happens

Tested in: Firefox Debeloper Edition 123.0b6 x64
Devtools console prints this error:
`Uncaught TypeError: e.split is not a function`

**Comment by boldenburg** [comment]
@NesCafe62 I don't know if you're still playing Screeps, but I also encountered this bug and managed to make a quick and dirty fix. You have to modify https://github.com/admon84/screeps-room-planner/blob/main/src/components/actions/ImportFromJson.tsx as follows:

https://github.com/admon84/screeps-room-planner/blob/8fd0fba347c85af11b7881760ce77d95dd21c617/src/components/actions/ImportFromJson.tsx#L4
change to
```js
import { Point, RoomStructuresJson } from '@/types';
```

https://github.com/admon84/screeps-room-planner/blob/main/src/components/actions/ImportFromJson.tsx#L62-L68
update this function to
```js
    Object.entries(json.structures).forEach(([structure, positions]) => {
      positions.forEach((pos) => {
        let point = pos as Point;
        let shortPoint = point.x + "-" + point.y;
        const tile = getTileForShort(shortPoint);
        addTileStructure(tile, structure);
        addStructurePosition(structure, shortPoint);
      });
    });
```

The bug is that the JSON import functionality assumes that the string it's parsing is in the custom shorthand notation the repo author chose to use in https://github.com/admon84/screeps-room-planner/blob/main/src/utils/constants.ts#L185.

**Comment by NesCafe62** [comment]
yep, I'm still playing screeps. good that you managed to find the correct fix. though I just used online version hosted here https://screeps-room-planner.vercel.app/ maybe we could sumbit a pull request with this fix

**Comment by ItsGoldeneyes** [comment]
I'm still running into the same error. Making a pull request!
